### DISCUSSION TITLE
Rollups - Time stamp 16:46 (Lesson 10: Blockchain Overview)

### DISCUSSION THREAD

**Discussion by ljb630** [Cyfrin/foundry-full-course-cu#1530] [Q&A] [answered]
As you mentioned, it solves scalability issues. Can you give some examples of how it affects L1?

**Comment by EngrPips** [comment, marked as answer]
Hello @ljb630 , this article might be a good read for your curiosity  https://hacken.io/discover/l1-l2-scalability/

**Reply by ljb630** [reply]
Thank you @EngrPips

**Reply by EngrPips** [reply]
Anytime friend.
